### DISCUSSION TITLE
Content under posts and above footer?

### DISCUSSION THREAD

**Discussion by mave-git** [GetPublii/Publii#1457] [Help]
Hi,
I tried to get on some themes such a part above the footers copyright text like many websites or like on getpublii.come. Don't see the way how to reach this.

Thanks!

![Bildschirmfoto vom 2024-03-23 14-54-11](https://github.com/GetPublii/Publii/assets/65040899/824e5d7e-635d-4f5c-b1a8-e6a79fc3d9e6)

**Comment by bobmitro** [comment]
This section is built in with five menu positions. 

Put the following code into the **footer.hbs** file _(Documents ▸ ⁨Publii⁩ ▸ ⁨sites⁩ ▸ ⁨YOUR_SITE ▸ ⁨input⁩ ▸ ⁨themes⁩ ▸ ⁨YOUR_THEME ▸ partials)_:
```
{{#if menus.footerMenu}}
   <nav>
      <ul class="footer__menu">
         {{> menu-footer menus.footerMenu}}
      </ul>
   </nav>
{{/if}}
```
Then, within the "partials" catalog, create the file named **menu-footer.hbs** with the following content:
```
{{#each items}}
   <li{{menuItemClasses}}>
      {{#if link}}
         <a href="{{menuUrl}}" {{#if title}} title="{{title}}" {{/if}}{{#if target}} target="{{target}}"{{/if}}{{#if rel}} rel="{{rel}}"{{/if}}>
            {{label}}
         </a>
         {{else}}
         <span{{#if title}} title="{{title}}"{{/if}}>
            {{label}}
         </span>
      {{/if}}
      {{#if items}}
         <ul>
            {{> menu-footer}}
         </ul>
      {{/if}}
   </li>
{{/each}}
```
Now, we have to register the footer menu position in our theme config.json file:
```
"menus": {
   "mainMenu": "Main menu",
   "footerMenu": "Footer menu"
},
```
**How it works?** 
The first level of the menu consists of a Text Separator, and the menu items are the child items. Additionally, you will need to style it using CSS.

<img width="1748" alt="screenshot 2024-03-23 at 15 42 36" src="https://github.com/GetPublii/Publii/assets/1391007/d23b64fb-7362-4608-b562-9e8d3109f852">

**Note:**
To preserve your custom code and prevent it being overwritten the next time the theme is updated, please ensure that you have created override files; these are, essentially, copies of the existing theme files that are stored separately from the main theme files → https://getpublii.com/dev/theme-overrides/

**Comment by mave-git** [comment]
Hi bobmitro,
thanks for sharing the code, but: It's titillating. I tried it for the theme Melissa. In config.json exists an entry "footerMenu"... But that's not the main thing. This is sobering and annoying: I purchased die theme-bundle to now determine that this are nice designs, but not real more functional. A more-using footer like asked is in GDPR-area a must-have. I thought this was easy to get via a post-entry, or pre-defined as a standard (!)
To get this, I've to code inside the system and at the end, I'm told dealing with CSS. No. Some colors are changeable in the gui, some not and I've to go special knowing not why decisions are made. So, buying the themes was not the goal and should be refund. If I must programming html, css for stancdard features, Publii was a mistaken choice.
Regards

**Comment by bobmitro** [comment]
Publii is not a "page builder"; the theme options are consistent across all themes, which ensures a seamless user experience and can be explored by installing any free themes. 

Furthermore, before committing to a purchase, customers can preview the theme on our demo server. We offer digital media (software) that becomes available for download instantly once a purchase is finalized. Due to the nature of digital sales, we do not offer refunds, as there is no way to return a digital product. 

It is important to highlight that information regarding our no-refund policy is clearly outlined on the checkout page, allowing customers to familiarize themselves with it before completing a purchase.

**Comment by mave-git** [comment]
Dear bobmitro,
this may be clear, and i wrote "should" refund. I knowing the policy and I mentioned the paying also as supporting the project and the idea. Yes, no "page builder" but there is another impression giving, when saying: 

> "Creating a website doesn't have to be complicated or expensive. With Publii, the most intuitive static site CMS, you can create a beautiful, safe, and privacy-friendly website quickly and easily; perfect for anyone who wants a fast, secure website in a flash."

Anyway, I'll give it further a try understanding the Publii-system.

**Comment by bobmitro** [comment]
> "Creating a website doesn't have to be complicated or expensive. With Publii, the most intuitive static site CMS, you can create a beautiful, safe, and privacy-friendly website quickly and easily; perfect for anyone who wants a fast, secure website in a flash."

That's right. Unlike other systems, you only need to write posts here, and the page builds itself based on your posts. Of course, if you want to customize the theme heavily, you will probably need to dig into its code, but due to the use of handlebars and CSS, it is not difficult.

PS. Send me via a ticket system (premium support) your full site backup, I will help you with the footer menus.

**Comment by miditaur** [comment]
Thanks for the detailed instructions!
